kingpin/actors/rightscale/deployment.py

```diff
@@ -98,13 +107,43 @@ def _execute(self):
             self.log.info('Would create a deployment %s' % self.option('name'))
             self.log.debug('Deployment params: %s' % params)
             raise gen.Return()
-        self.log.info('Deployment params: %s' % params)
 
         try:
             yield self._client.create_resource(
                 self._client._client.deployments, params)
         except Exception as e:
```

Comment:
Go ahead and import the requests library,... its getting loaded up anyways.
:)

Matt Wise
Sr. Systems Architect
Nextdoor.com

On Fri, Aug 21, 2015 at 10:26 AM, Mikhail Simin <notifications@github.com>
wrote:

> In kingpin/actors/rightscale/deployment.py
> <https://github.com/Nextdoor/kingpin/pull/282#discussion_r37656868>:
>
> > @@ -98,13 +107,43 @@ def _execute(self):
> >              self.log.info('Would create a deployment %s' % self.option('name'))
> >              self.log.debug('Deployment params: %s' % params)
> >              raise gen.Return()
> > -        self.log.info('Deployment params: %s' % params)
> >
> >          try:
> >              yield self._client.create_resource(
> >                  self._client._client.deployments, params)
> >          except Exception as e:
>
> I'd have to catch requests.http.exceptions and would import requests just
> for that... that's ok?
>
> Any way to configure rightscale.settings.RETRYING_SETTINGS similar to our
> tornado retry? To raise a specified exception for a matched string?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Nextdoor/kingpin/pull/282/files#r37656868>.
>